auth-service/src/main/kotlin/uk/gov/communities/delta/auth/Injection.kt

```diff
@@ -73,6 +79,21 @@ class Injection (
     val ssoLoginStateService = SSOLoginSessionStateService()
     val ssoOAuthClientProviderLookupService = SSOOAuthClientProviderLookupService(azureADSSOConfig, ssoLoginStateService)
     val microsoftGraphService = MicrosoftGraphService()
+    val cloudWatchMeterRegistry = CloudWatchMeterRegistry(
```

Comment:
We could check if `AUTH_METRICS_NAMESPACE` is set and use a SimpleMeterRegistry here instead if not

Reply:
Done :) Please could you check it again since it seems to always work for me locally 🙄

Reply:
That works 🙂